### PR TITLE
V3 doc

### PR DESCRIPTION
**Pull request by dabingnn**

### CODE REVIEW

File: cocos/math/MathUtil.h [hunk 1]
```diff
@@ -17,6 +17,25 @@
  Original file from GamePlay3D: http://gameplay3d.org
 
  This file was modified to fit the cocos2d-x project
+ * @~chinese 
+ * 黑莓公司2013年版权所有。
```
Comment (VisualSJ) [inline cocos/math/MathUtil.h]: 这个不要翻译吧

Comment (dabingnn) [inline cocos/math/MathUtil.h]: Thanks, I forgot it.

**Comment by zilongshanren** [comment]
@dabingnn  Is this PR reviewed?

**Comment by dabingnn** [comment]
@VisualSJ This pull request is updated, could you please review it?
Thanks

**Pull request merged**